config/tests/guest/libvirt/backuprestore.cfg

```diff
@@ -0,0 +1,214 @@
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.no_snapshot_attr.live_memspec.compress_format.bzip2
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.no_snapshot_attr.no_metadata_with_memspec
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.no_snapshot_attr.reuse_external
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.no_snapshot_attr.multi_disk_external
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.disk_only_spec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.check_libvirtd_log 
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.quiesce_with_diskonly.with_diskspec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_default
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.raw
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.xz
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.lzop
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.gzip
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.live_memspec.compress_format.bzip2
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.no_metadata_with_memspec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.reuse_external
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.multi_disk_external
+        #Need to fix the test case,
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.name_with_double_dash
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.quiesce_with_diskonly.no_diskspec
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.atomic_with_diskonly
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.no_metadata
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.no_metadata_with_diskonly
+        no virsh.snapshot_create_as.positive_tests.acl_test.file_disk.no_snapshot_attr.multi_snapshots
+        no virsh.snapshot_create_as.positive_tests.non_acl.file_disk.with_snapshot_attr.disk_only_spec
+        no virsh.snapshot_create_as..quiesce_without_diskonly #Not supported- guest agent
+        no virsh.snapshot_create_as.positive_tests.disk_only_spec #Needs fix, hardcoded as vda
+        no virsh.snapshot_create_as.positive_tests.reuse_external #Needs fix, hardcoded as vda
```

Comment:
I hope these are internal comments, let remove them and have it locally tracked and enable these tests with furture PRs..